susecloud

```diff
@@ -285,6 +285,16 @@ else
 fi
 
 #############################################################
+section_header "Ceph report"
+
+if [ -e /usr/bin/ceph ]; then
+    ceph_report_dir=$(mktemp -d)
```

Comment:
Actually I was more objecting to creating something in `/var/log` and then deleting it shortly after.  But that's a very good point about `split-supportconfig`, so yes maybe it's worth either tolerating it or dumping in `/var/log/supportconfig`.